Makefile

```diff
@@ -72,7 +72,8 @@ SRCS = \
 	menu/views/settings_editor.c \
 	menu/views/rtc.c \
 	menu/views/flashcart_info.c \
-	utils/fs.c
+	utils/fs.c \
+	menu/history.c
```

Comment:
Suggest you rename the file and this entry to something like `rom_history` and note that you should place them in alphabetical order.